lib/schema/schema.js

```diff
@@ -22,7 +22,6 @@ var amoeba = require('amoeba');
 var semver = require('semver');
 var except = amoeba.except;
 var util = require('util');
-
```

Comment:
This file can be reverted as there are no non-whitespace changes. Unless, of course, if this is needed to stop the editor from continually "fixing" the file.